zproject/backends.py

```diff
@@ -566,6 +567,24 @@ def authenticate(self, *, username: str, password: str, realm: Realm,
                      return_data: Optional[Dict[str, Any]]=None) -> Optional[UserProfile]:
         return None
 
+class PopulateUserLDAPError(Exception):
+    pass
```

Comment:
Should this be a subclass of ZulipLDAPException?

Reply:
@timabbott indeed, that seems better, I pushed again with this change